pytests/src/comparisons.rs

```diff
@@ -1,7 +1,8 @@
 use pyo3::basic::CompareOp;
 use pyo3::prelude::*;
+use std::fmt;
 
-#[pyclass]
+#[pyclass(frozen)]
```

Comment:
Any particular reason to add `frozen` to all of these? I'm definitely keen to make it the default soon 😄

Reply:
I needed to add `frozen` to derive `hash` on `OrderedDerived` so I added it everywhere to not give an edge to `OrderedDerived` in the benchmark.